netlify.toml

```diff
@@ -14,13 +14,14 @@
   command = "yarn build" # debug with: "yarn build --verbose"
 
   # Directory with the serverless Lambda functions to deploy to AWS.
-  functions = "public/lambda"
+  functions = "public/functions"
```

Comment:
Do we want to change this? Would we want to rename the directory in the file system also? Or is this a Gatsby v4 thing?

Reply:
All our functions are still in `/src/lambda/` currently.

Reply:
Yes, this is something we need to change due that we now deploy them from GC. More details here: https://support.gatsbyjs.com/hc/en-us/articles/360054529274-Deploying-Netlify-Functions-from-Gatsby-Cloud

Our source functions still live in `src/lambda`, what has changed is the destination of the compiled functions, `public/functions`. And that destination is the default destination used by Netlify.

Note: I tried keeping this untouched but Netlify wasn't able to find the functions when deployed from GC. It seems that this configuration only works in their CI/CD pipeline (when you link a GH repo and run the builds in their platform). 

Note 2: Besides that, I'm not sure even if the `netlify.toml` is being uploaded from GC. I have the feeling that only the content of the `/public` folder is transferred and Netlify only works as a plain hosting service.